docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
How do you guys feel about creating a `docker/bin` folder with some binstubs for `msfconsole`, and `msfvenom`? something like 
```bash
#! /bin/bash

# determine script path
pushd $(dirname $(dirname $(dirname $(realpath $0)))) > /dev/null
MSF_PATH=$(pwd)
popd > /dev/null

cd $MSF_PATH
docker-compose run --rm --service-ports ms ./msfvenom "$@"
```
Then users can just symlink them to their `$PATH`, and enjoy short commands.